### ISSUE TITLE
Incorrect missing file error

### ISSUE THREAD

**Issue by bjhardcastle** [AllenNeuralDynamics/aind-ephys-pipeline-kilosort25#10]
output:
```python
Command error:
  [capsule-7106853] cloning git repo...
  Cloning into 'capsule-repo'...
  [capsule-7106853] running capsule...
  + python -u run_capsule.py
  NWB backend: zarr
  Traceback (most recent call last):
    File "/tmp/nxf.KiTisPSf3L/capsule/code/run_capsule.py", line 155, in <module>
      sync_times = np.load(stream_folder / "synchronized_timestamps.npy")
    File "/opt/conda/lib/python3.9/site-packages/numpy/lib/npyio.py", line 427, in load
      fid = stack.enter_context(open(os_fspath(file), "rb"))
  FileNotFoundError: [Errno 2] No such file or directory: '../data/ecephys_session/ecephys_clipped/Record Node 104/experiment1/recording1/continuous/Neuropix-PXI-100.ProbeD-AP/synchronized_timestamps.npy'
```

This originates from here:
https://github.com/AllenNeuralDynamics/NWB_Packaging_Units/blob/main/code/run_capsule.py#L150-L156

- the code makes the false assumption that this is OpenEphys < 0.6, because it can't find `(stream_folder / "sample_numbers.npy")`
- the actual problem is that `stream_folder` doesn't exist:
  - `experiment1/recording1` doesn't exist
  - `experiment1/recording2` exists and should be used

A similar bit of code in SpikeInterface is also causing issues with aind-data-transfer uploads (Carter P might have been in contact about that):
https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/extractors/neoextractors/openephys.py#L247

**Comment by bjhardcastle** [comment]
This is the same issue as https://github.com/SpikeInterface/spikeinterface/pull/2655

**Comment by alejoe91** [comment]
Fixed by 58e0673b498a467a26aea12e60cd332db3a9ad33

**Comment by alejoe91** [comment]
Fixed here https://github.com/AllenNeuralDynamics/NWB_Packaging_Units/commit/74469e70b87ed4d604f5ff9432078fff902d2a3d
and in `aind-data-transfer`: https://github.com/AllenNeuralDynamics/aind-data-transfer/pull/329
